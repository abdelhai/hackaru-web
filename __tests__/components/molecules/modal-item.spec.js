import Factory from '@/__tests__/__setups__/factory';
import ModalItem from '@/components/molecules/modal-item';

describe('ModalItem', () => {
  it('render correctly', () => {
    const wrapper = new Factory(ModalItem, {
      slots: {
        default: '<p>Content</p>'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
