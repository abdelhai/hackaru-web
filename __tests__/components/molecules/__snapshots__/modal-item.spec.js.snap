// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalItem render correctly 1`] = `
<div
  class="modal-item"
>
  <p>
    Content
  </p>
</div>
`;
